.drone.yml

```diff
@@ -24,8 +24,7 @@ steps:
   pull: always
   settings:
     image_name: java11-mvn:${DRONE_COMMIT_SHA}
-    # a critical CVE is being whitelisted to get a release out.
-    whitelist: CVE-2019-5827, CVE-2021-25215, CVE-2021-26291, CVE-2021-27219, CVE-2021-25217 
+    whitelist: CVE-2019-5827, CVE-2020-14363, CVE-2021-25215, CVE-2021-27219, CVE-2021-31535, CVE-2019-5827, CVE-2021-37714
```

Comment:
I believe you should be able to get rid of CVE-2020-14363, CVE-2021-25215, CVE-2021-27219, CVE-2021-31535,  as it they aren't in the list of vulnerabilities (https://drone-gh.acp.homeoffice.gov.uk/UKHomeOffice/docker-java11-mvn/55/1/5). also CVE-2019-5827 is whitelisted twice

Reply:
@DJamesHO This has been resolved. I have removed the CVEs mentioned above and duplicate CVE.